### ISSUE TITLE
Chrome table content is not announced in NVDA

### ISSUE THREAD

**Issue by Maetz007** [nvaccess/nvda#9683]
### Steps to reproduce:
1. Use Chrome browser and ensure NVDA is loaded and running properly.
2. Go to: https://www.leaderscosmeticsusa.com/collections/aquablend-oil-pocket-masks/products/aquablend-oil-pocket-moisture-mask
3. Navigate to the "Shipping and Returns" tab.
4. Attempt to navigate into the table under the "We offer FREE GROUND SHIPPING to domestic customers on orders over $50 (before tax)." text with NVDA.

### Actual behavior:
NVDA does not announce the presence of a table

### Expected behavior:
NVDA announces the presence of a table and I can use NVDA's table commands to navigate the content of this table.

#### NVDA version:
2019.1.1

#### Windows version:
Windows 10 1803

#### Name and version of other software in use when reproducing the issue:
N/A

#### Does the issue still occur after restarting your PC?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes. The previous version of NVDA (2019.1) was able to see this table. In other words, NVDA 2019.1 read this content as a table, I updated NVDA, then went back to this same table in the same browser window and the table was no longer visible to NVDA 2019.1.1.

**Comment by DrSooom** [comment]
@Maetz007: Could you please try to reproduce this with [NVDA 2019.2 Beta build 17522](https://ci.appveyor.com/api/buildjobs/pksyej8s246fbl6g/artifacts/output/nvda_snapshot_beta-17522,a7bdcaea.exe) (newer than Beta 1) as well? Just run the Installer (temporary copy of NVDA). You don't have to install this Beta version on your system.
CC: @michaelDCurran

**Comment by Adriani90** [comment]
This is also reproducible on last NVDA Alpha.
alpha-17533,549a6aaa

I tried to enable layout tables in NVDA's settings under browse mode. After that, NVDA recognized that there is a table, but when I press the table navigation keys, I get following error in the log:

```
ERROR - scriptHandler.executeScript (19:32:42.255):
error executing script: <bound method ChromeVBuf.script_nextRow of <NVDAObjects.IAccessible.chromium.ChromeVBuf object at 0x05241290>> with gesture u'Alt+Strg+Pfeiltaste nach unten'
Traceback (most recent call last):
  File "scriptHandler.pyc", line 192, in executeScript
  File "documentBase.pyc", line 170, in script_nextRow
  File "documentBase.pyc", line 163, in _tableMovementScriptHelper
  File "virtualBuffers\gecko_ia2.pyc", line 331, in _getTableCellAt
LookupError
```

@michaelDCurran I guess this is a regression, and I think it could impact many tables in Chrome. AS far as I understand, the user did not have this issue in NVDA 2019.1 but the issue occurs since NVDA 2019.1.1.
cc: @leonardder

**Comment by Adriani90** [comment]
This works properly in NVDA alpha-20220,1c4519a6 and Chrome 83. Closing as works for me.
